contrib/charts/cert-manager/README.md

```diff
@@ -49,10 +49,12 @@ The following tables lists the configurable parameters of the cert-manager chart
 | `image.repository`     | Image repository                        | `quay.io/jetstack/cert-manager-controller`     |
 | `image.tag`            | Image tag                               | `v0.2.3`                                       |
 | `image.pullPolicy`     | Image pull policy                       | `Always`                                       |
+| `image.pullSecret`     | Image pull Secrets                      | `[]`                                           |
 | `replicaCount`         | Number of cert-manager replicas         | `1`                                            |
 | `createCustomResource` | Create CRD/TPR with this release        | `true`                                         |
 | `rbac.enabled`         | Create RBAC resources with this release | `true`                                         |
 | `resources`            | CPU/Memory resource requests/limits     | `None`                                         |
+| `customDNSServers`     | Custom DNS Servers to use               | `None`                                         |
```

Comment:
I'd remove this as a field here, and instead users can specify `--extra-args` (i.e. `extraArgs` in the Helm chart) for this.

I'd then also switch the PR to *not* load the resolv.conf from a file, but instead add a new flag to cert-manager `--acme-recursive-dns-resolvers` or the like, that accepts a comma separated list of nameservers. We've not used a config file elsewhere yet, and I don't think this feature warrants the addition.